app/src/main/res/values/strings.xml

```diff
@@ -65,6 +65,11 @@
     <string name="confirmOpenExternalApp">Would you like to leave DuckDuckGo to view this content?</string>
     <string name="unableToOpenLink">Unable to open this type of link</string>
 
+    <!-- Opening app links -->
```

Comment:
these strings should be in the `strings-untranslated.xml` so we can translate them in the next round.